### PR TITLE
Add support for Ubuntu 18.04

### PR DESCRIPTION
**Pull request by manics**
This uses https://github.com/ome/zeroc-ice-ubuntu1804 to install the Ice binaries.

If we decide to merge https://github.com/ome/zeroc-ice-ubuntu1804/pull/3 these paths need to be updated.

### CODE REVIEW

**Comment by jburel** [comment]
Note: The PR mentioned in the description has been merged

File: molecule/ice36-ubuntu1804/tests/test_default.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+@pytest.mark.parametrize('binary', [
+    'glacier2router',
+    'icegridadmin',
+    'icegridnode',
+])
+def test_ice_version(host, binary):
+    assert host.exists('icegridnode')
+    c = host.run('icegridnode --version')
+    assert c.rc == 0
+    assert c.stderr.startswith('3.6.')
```
Comment (jburel) [inline molecule/ice36-ubuntu1804/tests/test_default.py]: 3.6.5 maybe?

File: tasks/redhat.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  yum:
+    name: ice-all-devel
+    state: present
+  when: ice_install_devel
+
+- name: zeroc ice | install ice pip dependencies
+  become: true
+  yum:
+    name:
+      - gcc
```
Comment (jburel) [inline tasks/redhat.yml:27]: We have a different set of dependencies on omero-install (some probably from before we made the switch to https://github.com/ome/zeroc-ice-ubuntu1804...
Something to check. I can reduce to match that set and see what we got

File: molecule/ice36-ubuntu1804/tests/test_default.py (hunk 1 above)
Comment (manics) [inline molecule/ice36-ubuntu1804/tests/test_default.py]:
```suggestion
    assert c.stderr.startswith('3.6.5')
```

File: tasks/redhat.yml (hunk 2 above)
Comment (manics) [reply tasks/redhat.yml:27]: This is unchanged from the previous version of the role, it's just that it's been moved to a separate file

**Review by manics — COMMENTED** [review]
This is already up to date with the merged PR (uses release `0.3.0`)

**Comment by mtbc** [comment]
Given https://github.com/ansible/ansible/issues/19321 I am curious as to how the pip module name `zeroc-ice>=3.6,<3.7` works with `min_ansible_version: 2.6`.

**Comment by manics** [comment]
This is existing code already in production that's been working with Ansible 2.6 for ages, it's just been moved to a different file.

**Comment by jburel** [comment]
Merging so we are step closer to have deployment on Ubuntu with ansible

**Pull request merged**